src/alpha-lab/gaze_contingency_assistive.md

```diff
@@ -0,0 +1,72 @@
+---
+title: A practical guide to implementing gaze contingency in assistive technology
+description: "Gaze contingent systems for assistive technology"
+permalink: /alpha-lab/gaze-contingency-assistive
+tags: [Neon]
+---
+# A practical guide to implementing gaze contingency in assistive technology
```

Comment:
In assistive technology or
for assistive technology?

Reply:
Should be for